lib/robots/sdr_repo/preservation_ingest/update_catalog.rb

```diff
@@ -50,7 +50,13 @@ def remove_deposit_bag
         # Ceph backed storage and our use of hardlinking (instead of e.g. copying) to get content from the
         # deposit bag to the new Moab version.  see https://github.com/sul-dlss/preservation_catalog/issues/1633
         def rm_deposit_bag_safely_for_ceph
-          deposit_bag_pathname.rmtree
+          if deposit_bag_pathname.exist?
+            deposit_bag_pathname.rmtree
+          else
+            Honeybadger.notify("Deposit bag was missing. This is unusual; it's likely that the workflow step ran once before, and " \
+                               "failed on the network call to preservation_catalog. Please confirm that all is well with #{druid}.")
```

Comment:
I know this is already merged, but I'd suggest more specific guidance on what it means to "confirm that all is well". I have no idea what I would do.

And if we can't provide specific guidance, then we should remove the HB alert.

Reply:
good point!  attempted to remedy that in https://github.com/sul-dlss/preservation_robots/pull/467